### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by AnthonyLaw**

### CODE REVIEW

**Comment by fboucquez** [comment]
For rest apis, we should use open API, the generated docs. 

https://github.com/symbol/statistics-service/pull/75

https://fboucquez.github.io/statistics-service/openapi/index.html

I'm also using the open API to generate rest clients. (at least until we fully migrate to graphql in a future)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+To start the process, the task uses the nodes maintained by NEM Group.
+
+Once the whole list of nodes is known, additional information is gathered from each one of them such as `/node/info` and `/chain/info`.
+
+When the search completes, all node information is updated on the `Node` collection.
+
+### Chain Height Monitor Task
+
+To prevent chain forks happening, the finalization height from each node is gathered and height statistics are produced.
```
Comment (segfaultxavi) [inline README.md:53]: What do you mean? How does the stats service prevent any chain fork?

**Review by segfaultxavi — APPROVED** [review]
Very very nice!

Just one question, though. Except for that, this looks good to me.

File: README.md (hunk 1 above)
Comment (AnthonyLaw) [reply README.md:53]: If we found some nodes with the wrong chain height, we can let the node owner resync or update the latest version.
can it consider as prevent?

**Pull request merged**